programmers/크레인-인형뽑기-게임.js

```diff
@@ -0,0 +1,19 @@
+function solution(board, moves) { 
+    const basket = [];
+    let count = 0;
+    for (let el of moves) {
+        for (let i = 0; i < board.length; i++) {
+            if (board[i][el - 1] !== 0) {
+                basket.push(board[i][el - 1]);
+                board[i][el - 1] = 0;
+                break; 
+            }
+        }
+        if (basket.length > 1 && basket[basket.length - 1] === basket[basket.length - 2]) {
+            basket.pop()
+            basket.pop()
```

Comment:
pop을 두번 쓰지 않고 이런식으로 한번에 하는것이 가능합니다.
```suggestion
basket.splice(basket.length - 2, 2);
```